src/odb/src/db/dbAccessPoint.h

```diff
@@ -58,7 +58,8 @@ class _dbAccessPoint : public _dbObject
  public:
   _dbAccessPoint(_dbDatabase*, const _dbAccessPoint& r);
   _dbAccessPoint(_dbDatabase*);
-  ~_dbAccessPoint();
+
+  ~_dbAccessPoint() = default;
```

Comment:
Don't know if this applies here: this destructor is not virtual, so if there are any subclasses of this class, there could be a memory leak since the destructor of the subclass won't be run.

Reply:
There is no change in that regard. There classes are not used with inheritance